src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -0,0 +1,812 @@
+          {{/menu-item-text}}
+        {{/menu-item-main}}
+      {{/menu-item}}
+    {{/menu-list-item}}
+    {{#> menu-list-item}}
+      {{#> menu-item}}
+        {{#> menu-item-main}}
+          {{#> menu-item-text}}
+            Action 2
+          {{/menu-item-text}}
+        {{/menu-item-main}}
+      {{/menu-item}}
+    {{/menu-list-item}}
+    {{#> menu-list-item}}
+      {{#> menu-item}}
+        {{#> menu-item-main}}
+          {{#> menu-item-text}}
+            Action 3
+          {{/menu-item-text}}
+        {{/menu-item-main}}
+      {{/menu-item}}
+    {{/menu-list-item}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### With-links
+```hbs
+{{#> menu}}
+  {{#> menu-body}}
+    {{#> menu-list-item}}
+      {{#> menu-item menu-item--type="a" menu-item--attribute='href="#"'}}
+        {{#> menu-item-main}}
+          {{#> menu-item-text}}
+            Link 1
+          {{/menu-item-text}}
+          {{> menu-item-external-icon}}
+        {{/menu-item-main}}
+      {{/menu-item}}
+    {{/menu-list-item}}
+    {{#> menu-list-item}}
+      {{#> menu-item menu-item--type="a" menu-item--attribute='href="#"'}}
+        {{#> menu-item-main}}
+          {{#> menu-item-text}}
+            Link 2
+          {{/menu-item-text}}
+          {{> menu-item-external-icon}}
+        {{/menu-item-main}}
+      {{/menu-item}}
+    {{/menu-list-item}}
+    {{#> menu-list-item}}
+      {{#> menu-item menu-item--type="a" menu-item--attribute='href="#"'}}
+        {{#> menu-item-main}}
+          {{#> menu-item-text}}
+            Link 3
+          {{/menu-item-text}}
+          {{> menu-item-external-icon}}
+        {{/menu-item-main}}
+      {{/menu-item}}
+    {{/menu-list-item}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### With-separator(s)
+```hbs
+{{#> menu}}
+  {{#> menu-body}}
+    {{#> menu-list-item}}
+      {{#> menu-item}}
+        {{#> menu-item-main}}
+          {{#> menu-item-text}}
+            Action 1
+          {{/menu-item-text}}
+        {{/menu-item-main}}
+      {{/menu-item}}
+    {{/menu-list-item}}
+    {{#> menu-list-item}}
+      {{#> menu-item}}
+        {{#> menu-item-main}}
+          {{#> menu-item-text}}
+            Action 2
+          {{/menu-item-text}}
+        {{/menu-item-main}}
+      {{/menu-item}}
+    {{/menu-list-item}}
+    {{> divider}}
+    {{#> menu-list-item}}
+      {{#> menu-item}}
+        {{#> menu-item-main}}
+          {{#> menu-item-text}}
+            Action 3
+          {{/menu-item-text}}
+        {{/menu-item-main}}
+      {{/menu-item}}
+    {{/menu-list-item}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### With-titled-groups
+```hbs
+{{#> menu}}
+  {{#> menu-body}}
+    {{#> menu-list-item}}
+      {{#> menu-item menu-item--type="a" menu-item--attribute='href="#"'}}
+        {{#> menu-item-main}}
+          {{#> menu-item-text}}
+            Link not in group
+          {{/menu-item-text}}
+        {{/menu-item-main}}
+      {{/menu-item}}
+    {{/menu-list-item}}
+    {{> divider}}
+    {{#> menu-group}}
```

Comment:
this is a `section` and the parent is a `ul`

Reply:
based on conversation with matt, i think im going to update the body to use `<div>` and then you could nest multiple menu's inside of the body. that would fix this problem too.